docs/exchanges/binance.md

```diff
@@ -9,15 +9,19 @@ tags:
 
 ## 📁 [Connector folder](https://github.com/hummingbot/hummingbot/tree/master/hummingbot/connector/exchange/binance)
 
+## 📝 Signup
+
+* Hummingbot referral link - <https://www.binance.com/en/register?ref=FQQNNGCD>
+* Sign up with Binance using Hummingbot's referral link for a 10% discount!
```

Comment:
* Sign up with Binance using Hummingbot's referral link for a 10% discount off trading fees